src/charm.py

```diff
@@ -108,6 +107,24 @@ def __init__(self, *args):
         self.framework.observe(self.db.on.standby_changed, self._on_standby_changed)
         self.framework.observe(self.on.db_master_available, self.configure_pod)
 
+    def _on_grant_admin_role_action(self, event):
+        """Handle the grant-admin-role action."""
```

Comment:
Should the new functions being added comply with our standards? E.g., describe the arguments in the docstring?

Reply:
Apart from the licence/license change and the removal of the functions put in environment.py, I haven't changed the charm.py.  
I'll fix this when I'll opt-in operator-workflows